x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/store/ReservedRolesStore.java

```diff
@@ -54,7 +54,7 @@
                         null, null,
                         MetadataUtils.DEFAULT_RESERVED_METADATA, null))
                 .put("monitoring_user", new RoleDescriptor("monitoring_user",
-                        new String[] { "cluster:monitor/main", "cluster:monitor/xpack/info" },
+                        new String[] { "cluster:monitor/main", "cluster:monitor/xpack/info", "cluster:monitor/remote/info" },
```

Comment:
More of an observation, maybe we can use the static member `NAME` from action `RemoteInfoAction.NAME`. And, we can also do that for `cluster:monitor/main` and `cluster:monitor/remote/info`. Don't know if they're meant to be used like constants though